sr_robot_launch/launch/sr_hardware_control_loop.launch

```diff
@@ -18,14 +18,18 @@
   <arg name="arm_ctrl" default="$(arg arm)"/>
   <!-- Whether to run hand controllers. -->
   <arg name="hand_ctrl" default="$(arg hand)"/>
-  <!-- Set hand type. -->
+
   <arg name="hand_type" default="hand_e"/>
+  <arg name="hand_version" default="E3M5"/>
+  <arg name="fingers" default="th,ff,mf,rf,lf"/>
```

Comment:
does it matter if this is all or the specified?

Reply:
no, they are equivalent but left specificed for the user to understand the format